src/main/java/seedu/heymatez/model/assignee/Assignee.java

```diff
@@ -12,7 +12,8 @@
             + "proper name";
 
     public static final String MESSAGE_CONSTRAINTS = "Assignee specified must be a member in the "
-            + "displayed member's list, with the exact name (case-sensitive).";
+            + "displayed member's list, with the exact name (case-sensitive and format-sensitive) and name should not "
+            + "have extra spaces or characters.";
```

Comment:
I think for the extra spaces, can specify 'extra spaces in between 2 words' since extra spaces before/after the entire name is allowed.